src/components/table/table.tsx

```diff
@@ -387,8 +387,18 @@ export class Table {
     render() {
         return (
             <div id="tabulator-container">
-                <div id="tabulator-table" />
+                <div
+                    id="tabulator-table"
+                    class={{
+                        'has-pagination': this.totalRows > this.pageSize,
+                        'has-calculations': hasCalculations(this.columns),
+                    }}
+                />
             </div>
         );
     }
 }
+
+function hasCalculations(columns: Column[]): boolean {
```

Comment:
Shouldn't we write this function in the `Table`  class? And what do you think if we make it private function?
The other thing is I may be wrong but what this function does is to check if at least one of the columns has `aggregator`. What do you think if we name it `hasAggregator()`